### PR TITLE
Fix modular consoles

### LINKED ISSUE #28745: Engineering Modular Console Not Functioning
I noticed in game and on my offline code that the modular engineering console (one in the top right of power storage on boxstation) is non-functional, in that none of the modular components (hard drive, etc) spawn. 

Spawning in a new computer in game works, but the one's set down at round start don't seem to work.

### PR DESCRIPTION
**Pull request by p440**
processors try to add components to the list on Initialize()
list probably ~~doesnt exist yet due to vOv~~ got populated then paved over in the base Initialize()

Fixes #28745

:cl: drline
fix: Nanotrasen finally sent out IT personnel to plug your modular consoles back in. You're welcome.
/:cl:

### CODE REVIEW

File: code/modules/modular_computers/computers/item/computer.dm [hunk 1]
```diff
@@ -49,14 +49,15 @@
 	var/comp_light_color			//The color of that light
 
 
-
+/obj/item/device/modular_computer/New()
+	all_components = list()
```
Comment (GunHog) [inline code/modules/modular_computers/computers/item/computer.dm]: Why is this in `New()` and not simply defined with the `all_components` declaration? Should you not just have `var/list/all_components = list()` on line 42 and take it entirely away from `New()` and `Initialize()`?

Comment (Jordie0608) [inline code/modules/modular_computers/computers/item/computer.dm]: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#startupruntime-tradeoffs-with-lists-and-the-hidden-init-proc

Comment (GunHog) [inline code/modules/modular_computers/computers/item/computer.dm]: Oh...well...okay then...

**Review by Cyberboss — CHANGES_REQUESTED** [review]
This will only break it harder because you didn't call ..() in New. Anyway just create the list in the declaration rather than create a New because New is deprecated for atoms

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**